assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserDimensionsPieChart.js

```diff
@@ -160,12 +157,19 @@ export default function UserDimensionsPieChart( {
 			);
 
 			const othersLabel = __( 'Others', 'google-site-kit' ).toLowerCase();
-			if ( sourceLink && rowLabel === othersLabel ) {
-				tooltip += getTooltipHelp(
-					sourceLink,
-					__( 'See the detailed breakdown in Analytics', 'google-site-kit' ),
-					'others'
-				);
+			if ( rowLabel === othersLabel ) {
+				switch ( dimensionName ) {
+					case 'ga:country':
+						tooltip += `<p>${ __( 'See the full list of locations in Analytics', 'google-site-kit' ) }`;
+						break;
+					case 'ga:deviceCategory':
+						tooltip += `<p>${ __( 'See the full list of devices in Analytics', 'google-site-kit' ) }`;
+						break;
+					case 'ga:channelGrouping':
+					default:
+						tooltip += `<p>${ __( 'See the full list of channels in Analytics', 'google-site-kit' ) }`;
+						break;
+				}
 			}
 
 			if ( otherSupportURL && rowLabel === '(other)' ) {
```

Comment:
Yes, "(other)" and "(not set)" should still have their tooltip links, since those are reachable: When a pie chart includes one of these slices (only happens based on whether one of these dimension values is present in your GA report), the slices can be clicked. At that point, the tooltip is permanently visible, and the link can be clicked.

Reply:
Here's a screenshot of "(not set)", which is present in my Locations report:
<img width="528" alt="Screen Shot 2021-02-25 at 10 40 23 AM" src="https://user-images.githubusercontent.com/3531426/109200826-e69e6d80-7755-11eb-8bcf-2fc1b5b21a7c.png">